src/website/views/components/steps/_examples.pug

```diff
@@ -302,6 +302,257 @@ p.-text
+                            | Helpful information goes here.
+                    .chi-steps__subitem.-completed
+                      i.chi-icon.icon-check.-xs(aria-hidden="true")
+                      p Completed item 
+                        span.-text--2xs  | 17 Feb 2021
+                      button.chi-button.-icon.-xs.-flat.-p--0.-ml--1(id="example__help-button-2" aria-label="Help" data-target="#example__help-popover-2")
+                        i.chi-icon.icon-circle-info-outline.-icon--dark.-mr--0(aria-hidden="true")
+                      section.chi-popover.chi-popover--top(id="example__help-popover-2" aria-modal="true" role="dialog")
+                        .chi-popover__content
+                          p.chi-popover__text
+                            | Helpful information goes here.
+                    .chi-steps__subitem.-completed.-position--relative
+                      i.chi-icon.icon-arrow-right.-sm.-icon--warning.-position--absolute(aria-hidden="true", style="left:-48px;")
+                      i.chi-icon.icon-warning.-xs.-icon--warning(aria-hidden="true")
+                      p Completed item 
+                        span.-text--2xs  | 17 Feb 2021
+                      button.chi-button.-icon.-xs.-flat.-p--0.-ml--1(id="example__help-button-3" aria-label="Help" data-target="#example__help-popover-3")
+                        i.chi-icon.icon-circle-info-outline.-icon--dark.-mr--0(aria-hidden="true")
+                      section.chi-popover.chi-popover--top(id="example__help-popover-3" aria-modal="true" role="dialog")
+                        .chi-popover__content
+                          p.chi-popover__text
+                            | Helpful information goes here.
+                div.chi-steps__line
+            else if step == 'Step 3'
+              li.chi-steps__item
+                div.chi-steps__icon
+                  div.chi-steps__content
+                    a.chi-steps__item-title.-text--h5.-text--navy.-lh--2.-m--0(href='#')= `${step}`
+                    .chi-steps__subitem.-d--block.-pl--0
+                      .-d--flex.-justify-content--center.-align-items--center.-mb--1
+                        i.chi-icon.icon-warning.-xs.-icon--warning(aria-hidden="true")
+                        p Pending item 
+                          span.-text--2xs.-text--navy  | 17 Feb 2021
+                        button.chi-button.-icon.-xs.-flat.-p--0.-ml--1(id="example__help-button-4" aria-label="Help" data-target="#example__help-popover-4")
+                          i.chi-icon.icon-circle-info-outline.-icon--dark.-mr--0(aria-hidden="true")
+                        section.chi-popover.chi-popover--top(id="example__help-popover-4" aria-modal="true" role="dialog")
+                          .chi-popover__content
+                            p.chi-popover__text
+                              | Helpful information goes here.
+                      button.chi-button.-xs(style="margin-left: 1.125rem;") Schedule
```

Comment:
Can we create an action container (e.g. `chi-steps__actions`) to store buttons or links related to the step items and then create a css rule that targets action containers within `chi-steps__subitem` and apply the margin-left there versus defining inline?